saleor/graphql/order/types.py

```diff
@@ -231,12 +231,14 @@ class Meta:
     def resolve_thumbnail(root: models.OrderLine, info, *, size=None):
         if not root.variant_id:
             return None
-        if not size:
-            size = 255
         image = root.variant.get_first_image()
-        url = get_product_image_thumbnail(image, size, method="thumbnail")
-        alt = image.alt if image else None
-        return Image(alt=alt, url=info.context.build_absolute_uri(url))
+        if image:
```

Comment:
`size` can be also set to 255 as a default argument for function. This will remove one `if` from the function
```
def resolve_thumbnail(root: models.OrderLine, info, *, size=255):
```

If you invert `if` condition you will reduce one `if` injection. 
```
    image = root.variant.get_first_image()
    if not image:
        return None

    url = get_product_image_thumbnail(image, size, method="thumbnail")
    alt = image.alt
    return Image(alt=alt, url=info.context.build_absolute_uri(url))
```